cli/azd/test/functional/cli_test.go

```diff
@@ -189,7 +189,7 @@ func Test_CLI_InfraCreateAndDelete(t *testing.T) {
 	ctx = exec.WithCommandRunner(ctx, exec.NewCommandRunner(os.Stdin, os.Stdout, os.Stderr))
 
 	// GetResourceGroupsForEnvironment requires a credential since it is using the SDK now
-	cred, err := azidentity.NewDefaultAzureCredential(nil)
+	cred, err := azidentity.NewAzureCLICredential(nil)
```

Comment:
What's the difference between `NewAzureCLICredential` and `NewDefaultAzureCredential`?